demisto_sdk/commands/common/tools.py

```diff
@@ -820,6 +822,7 @@ def get_file(
     clear_cache: bool = False,
     return_content: bool = False,
     keep_order: bool = False,
+    raise_on_error: bool = False,
```

Comment:
great, please document (docstring) the fact that we used to just return empty on error, and now allow catching errors